pyshelf/search/connection.py

```diff
@@ -55,9 +55,6 @@ def _parse_url(self):
         """
         self._parsed_url = urlparse(self.connection_string)
 
-        if not self._parsed_url.scheme:
```

Comment:
will the scheme always exist?  It might in our case, just saying, it would be nice to guard against it in case we use it in another place in the future